shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -435,7 +465,15 @@ - (nullable id)accessibilityElementAtIndex:(NSInteger)index {
   if (index == 0) {
     return _semanticsObject;
   }
+
+  // Create an addtional child to act as the accessibility container for the platform view.
```

Comment:
nit: can you update this comment? It just returns an extra child and doesn't create it.